cabot/cabotapp/tasks.py

```diff
@@ -24,13 +28,34 @@
 logger = logging.getLogger(__name__)
 
 
-@task(ignore_result=True)
-def run_status_check(check_or_id):
-    if not isinstance(check_or_id, models.StatusCheck):
-        check = models.StatusCheck.objects.get(id=check_or_id)
+def _classify_status_check(pk):
```

Comment:
You never actually call _classify_status_check in this version (I know it was called before!)

Reply:
Ah! I did a rebase and while fixing a conflict accidentally removed it!